doc/api/debugger.md

```diff
@@ -179,12 +183,13 @@ process or via URI reference to the listening debugger:
 * `node debug <URI>` - Connects to the process via the URI such as
 localhost:5858
 
-## V8 Inspector Integration for Node.js
+### V8 Inspector Integration for Node.js
 
 **NOTE: This is an experimental feature.**
```

Comment:
Yeah, I was thinking about what the "least worst" option would be. It seemed also bad to remove the doc warning while the runtime still writes one ("docs say go ahead but then the runtime punishes the user with scary warnings").